plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/utils/KGraphMappingUtil.xtend

```diff
@@ -104,8 +106,21 @@ class KGraphMappingUtil {
         skedge.junctionPoints = new KVectorChain()
         skedge.junctionPoints.addAllAsCopies(0,kedge.getProperty(CoreOptions.JUNCTION_POINTS))
         skedge.junctionPoints.offset(new KVector(leftInset, topInset))
-        // Copy render scale.
-//        skedge.renderScale = kedge.getProperty(CoreOptions.TOP_DOWN_LAYOUT_RENDER_SCALE);
+
+        // TODO: not sure whether the above specially handled junction/routing points can be handled with the 
```

Comment:
@NiklasRentzCAU can you say anything about that?

Reply:
Yes, they cannot be handled via the properties, as they are no properties. This mapping is done because SGraph handles padding differently and belongs to the layout part of converting a KGraph to an SGraph.